server/controller/assessment/getSections.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseProtocol, DB } from '@server/db'
+import { AssessmentRepository } from '@server/repository'
+
+export const getSections = async (
+  props: { assessmentName: string; assessmentCycleName: string },
+  client: BaseProtocol = DB
+): Promise<any> => {
+  const { assessmentName, assessmentCycleName } = props
+
+  const assessmentCycle = await AssessmentRepository.readCycle({ assessmentCycleName }, client)
```

Comment:
this query can also be removed, you can read the assessment and then find the cycle